client/components/close-on-escape/test/index.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import React from 'react';
+import ReactDom, { unmountComponentAtNode } from 'react-dom';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import CloseOnEscape from '../';
+
+const simulateEscapeKeydown = () => (
```

Comment:
As an aside, I think the component with the most input testing would be https://github.com/Automattic/wp-calypso/blob/master/client/components/token-field/test/index.jsx if you'd like to see a more complicated example.

I'm not sure if we've updated recommendations there in terms of test setup.